crates/proof-of-sql/src/proof_primitive/mod.rs

```diff
@@ -1,4 +1,6 @@
 //! TODO: add docs
 pub mod dory;
+/// Central location for any code that is common to the hyrax and dynamic dory schemes.
+pub(super) mod dynamic_dory_and_hyrax_common_utils;
```

Comment:
NIT: See comment above about naming this module something more abstract to accommodate future commitment schemes